### DISCUSSION TITLE
Welcome to kiwix-desktop Discussions!

### DISCUSSION THREAD

**Discussion by kelson42** [kiwix/kiwix-desktop#839] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by steversp** [comment]
Will Kiwix Desktop 2.3.0 work on windows 7?  (My team have downloaded Kiwix 2.0 from kiwix.org but it will not work on Window 7)

**Reply by steversp** [reply]
If Kiwix Desktop 2.3.0 does not work on windows 7, is there a version of Kiwix Desktop that does?

**Reply by Popolechien** [reply]
Hi @steversp, where are your deployments taking place? 

There are no plans for Kiwix-desktop 2.x to be retroactively compatible with older Windows versions, but the [electron app](https://kiwix.github.io/kiwix-js-windows/kiwix-js-electron.html) should.

**Reply by kelson42** [reply]
@steversp what is ecactly the problem with your win7? please open a bug with all the details.

**Reply by steversp** [reply]
My team installed the electron app an it will display informational content on our local computer but it will be act a server to other computers including smartphones.  Our local computer is set to being a Wifi hotspot.  Is the "electron app" able act as a server to other computers?  If so, do you have link to the instructions for us to set it to do so?  I can ask my technician in Kabul specify what is not working with Kiwix Desktop on Windows7.

**Reply by Popolechien** [reply]
@steversp No, the electron app will not act as Kiwix-server. To open a bug ticket with all details, please go to https://github.com/kiwix/kiwix-desktop/issues
Add as much info as possible: type of computer, windows version, kiwix version, types of files, etc. etc. so we can try and reproduce on our end.

**Reply by steversp** [reply]
Ok.  Thanks I will ask them to provide those details.  Given below is some info about the computer: a) Computer Brand Name: hp
b) Year Made: 2013
c) Operating System Version: Windows 7
d) Amount of RAM Memory 8 GB
e) Hard-drive capacity 500 GB
f) Computer include MS Office software such as MS Word and MS Excel

**Comment by kelson42** [comment]
@steversp thx, don't foget to run it from the command line, and tell us what is written on the terminal/command line.
